tuxemon/resources/db/locale/es_MX.json

```diff
@@ -38,5 +38,7 @@
     "technique_bite": "Muerde",
     "technique_poison_sting": "Picadura Venenosa ",
     "technique_pound": "Libra",
-    "technique_water_shot": "Disparo De Agua"
+    "technique_water_shot": "Disparo De Agua",
+    "dont_care": "No importa.",
+    "default_names": ["Cruz","Loreto","Lupe","Chus","Cande","Ariel","Ale","Manu","Zorion","Reyes","Odalis","Nazaret"]
```

Comment:
Those are very unusual names (except Manu). 

- I have never met (or heard about) any Spanish person called "Cruz" (this is a surname), "Ariel", "Ale", "Zorion", "Reyes" (maybe in Mexico?), "Odalis" or "Nazaret".

- Some of the remaining ones are diminutives: "Lupe" is "Guadalupe" (uncommon nowadays), "Chus" is "María Jesús" (in this case I think "Chus" is better), "Cande" is "Candela"(also uncommon) and "Manu" is "Manuel".

- "Loreto" exists, but is uncommon too.